README.rst

```diff
@@ -27,38 +26,111 @@ Installation on Open edX Devstack
+1. Clone this repository.
 
 .. code-block:: bash
 
   git clone git@github.com:Pearson-Advance/openedx-lti-tool-plugin.git
 
-- Set up a virtualenv
+2. Set up a virtualenv.
 
 .. code-block:: bash
 
   cd openedx-lti-tool-plugin
   virtualenv venv
   source venv/bin/activate
 
-- Install development dependencies.
+3. Install development dependencies.
 
 .. code-block:: bash
 
   make dev-requirements
 
-Settings
-========
+LTI 1.3 Resource Launch Setup
+=============================
+
+1. (Optional) Expose the LMS service to an external domain.
+2. Go to LMS Admin > PyLTI 1.3 Tool Config > Lti 1.3 tools.
+3. Create a new LTI 1.3 tool configuration with all your platform details.
+4. Go to your LTI platform and set the login and keyset URL of the LTI tool:
+  - Login URL: https://lms-address.com/openedx_lti_tool_plugin/1.3/login
+  - Keyset URL: https://lms-address.com/openedx_lti_tool_plugin/1.3/pub/jwks
+5. Setup the tool link URL:
+  - Course Unit/Problem URL: https://lms-address.com/openedx_lti_tool_plugin/1.3/launch/course-id/unit-or-problem-id
+  - Complete Course URL (This requires the "Complete Course Launch" feature): https://lms-address.com/openedx_lti_tool_plugin/1.3/launch/course-id
+6. Execute an LTI 1.3 resource launch from the LTI platform.
+7. The LTI 1.3 resource launch should successfully take you to the requested content.
+
+LTI 1.3 AGS Compatibility
+=========================
+
+This LTI tool supports the AGS service for both unit/problem and course resource launches, the AGS service requires the LTI platform to send a line item and allow POST score updates, if no line item or POST score permission is sent, the resource launch will fail requesting such configuration.
+
+- Service Claims Example: https://www.imsglobal.org/spec/lti-ags/v2p0#example-link-has-a-single-line-item-tool-can-only-post-score
+
+Plugin Settings
+***************
+
+LMS Settings
+============
+
+- `OLTITP_ENABLE_LTI_TOOL`: Enables or disables the LTI tool plugin.
+
+Django Waffle Switches
+======================
+
+- `openedx_lti_tool_plugin.allow_complete_course_launch`: Toggles the "Course Access Configuration" feature.
+- `openedx_lti_tool_plugin.course_access_configuration`: Toggles the "Complete Course Launch" feature.
+- `openedx_lti_tool_plugin.save_pii_data`: Toggles the "Save PII Data" feature.
+
+Optional Features
+*****************
+
+Course Access Configuration
+===========================
+
+By default, all resource links to all courses are allowed. If course access needs to be restricted you can set up a course access configuration policy for each LTI tool configuration deployed, this will allow you to set a list of allowed courses that an LTI tool configuration deployed is allowed for LTI 1.3 resource launches. Follow these next steps to set this feature:
+
+1. Enable the Django Waffle switch: `openedx_lti_tool_plugin.course_access_configuration`.
+2. Go to LMS Admin > Open edX LTI Tool Plugin > Course access configurations.
```

Comment:
@kuipumu I am getting a 500 when going to https://partner-courses.pearson.com/admin/openedx_lti_tool_plugin/courseaccessconfiguration/

Reply:
@Jacatove I found the error in the logs, the LMS is missing migrations